test/jdk/javax/swing/JTree/6263446/bug6263446.java

```diff
@@ -166,9 +181,11 @@ private static void createAndShowGUI() {
 
 
         frame.getContentPane().add(tree);
-        frame.pack();
+        frame.setAlwaysOnTop(true);
```

Comment:
Java doc says its better to check whether the platform supports this method before calling it 
https://docs.oracle.com/javase/7/docs/api/java/awt/Window.html#setAlwaysOnTop(boolean)

Reply:
it also says "If always-on-top mode isn't supported for this window or this window's toolkit does not support always-on-top windows, calling this method has no effect."
and I dont see we check for the support in any of the swing tests, so I guess we are ok..